src/Features/Demo/DemoGhostEntity.cpp

```diff
@@ -56,7 +56,7 @@ void DemoGhostEntity::NextDemo()
         }
     } else {
         this->hasFinished = true;
-        this->sameMap = false; //Make the ghost to disapear
+        this->sameMap = false; //Make the ghost to dissapear
```

Comment:
correct spelling is 'disappear'

Reply:
Whoops I though the autocorrect had stopped complaining on that one, I must have been really tired